graph.py

```diff
@@ -89,12 +88,17 @@ class RawRepoData:
     _cache = TTLCache(100, ttl=_ttl)
     _last_expiry = time.monotonic()
 
-    def __init__(self, channel: str, arch: str = "linux-64", ttl=600):
+    def __init__(self, channel: str, arch: str = "linux-64", base_url: str = "https://conda.anaconda.org/", ttl=600):
         # setup cache
         self.ttl = ttl
         # normal seetings
         logger.info(f"RETRIEVING: {channel}, {arch}")
-        url_prefix = f"https://conda.anaconda.org/{channel}/{arch}"
+        if '{channel}' in base_url and '{arch}' in base_url:
+            url_prefix = base_url.format(channel=channel, arch=arch)
+        elif '{channel}' in base_url:
+            url_prefix = base_url.format(channel=channel) + f"/{arch}"
+        else:
+            url_prefix = f"{base_url}/{channel}/{arch}"
```

Comment:
double slashes here in the `conda.anaconda.org/` case

Reply:
Agreed, there used to be an rstip here